ts/ui/menu/Menu.ts

```diff
@@ -557,14 +557,18 @@ export class Menu {
     this.about.attachMenu(menu);
     this.help.attachMenu(menu);
     this.originalText.attachMenu(menu);
-    this.annotationText.attachMenu(menu);
     this.mathmlCode.attachMenu(menu);
     this.zoomBox.attachMenu(menu);
     this.checkLoadableItems();
     this.enableExplorerItems(this.settings.explorer);
-    menu.showAnnotation = this.annotationText;
-    menu.copyAnnotation = this.copyAnnotation.bind(this);
-    menu.annotationTypes = this.options.annotationTypes;
+    const cache = [];
```

Comment:
I had to make this

``` js
  const cache: [string, string][] = [];
```

to get it to compile without errors.

Reply:
Same here. Also solved.